jwt.test-d.ts

```diff
@@ -26,7 +26,7 @@ app.register(fastifyJwt, {
         authorizationTokenInvalid: (err) => `${err.message}`,
         authorizationTokenUntrusted: 'Token untrusted'
     },
-    trusted: () => true,
+    trusted: () => false || '' || Buffer.from('foo'),

```

Comment:
Is this the only test that is needed? Is this code only to check if `JWTTypes` are outside of the fastify namespace? Can you please add a test for those?

Reply:
> Is this the only test that is needed?

`boolean | Promise<boolean> | JWTTypes.SignPayloadType | Promise<JWTTypes.SignPayloadType>`
it missed the promise types, I'm going to add the promise types test

> Is this code only to check if JWTTypes are outside of the fastify namespace?

No, it doesn't check for `JWTTypes`, I don't think we need to check it.
It only fixes miss-reference to (unknown) JWTTypes from outside the namespace.